src/main/java/org/owasp/wrongsecrets/challenges/Difficulty.java

```diff
@@ -0,0 +1,14 @@
+package org.owasp.wrongsecrets.challenges;
+
+public enum Difficulty {
```

Comment:
Would it maybe work if we use a static class instead with 
static int EASY=1,
static int NORMAL=2,
static int HARD=3,
static int EXPERT=4
static int MASTER=5
static int MAX=5
so we don't have to put toInt in almost all the places?

Reply:
That wouldn't work with:

```
 public String minimal() {
            return "☆".repeat(difficulty.toInt());
        }
```
As you receive only the level not the exact value.

Reply:
If the challenge its difficulty value is an int, it should work right?